packages/playground/vue/ScanDep.vue

```diff
@@ -1,8 +1,15 @@
 <template>
   <h2>Scan Deps from &lt;script setup lang=ts&gt; blocks</h2>
   <div class="scan">{{ typeof debounce === 'function' ? 'ok' : 'error' }}</div>
+  <div class="multiline-scan">{{body}}</div>
 </template>
 
 <script setup lang="ts">
 import { debounce } from 'lodash-es'
+let body = ""
+function testMultiline(a, b) {
+  body = b.body
+}
+testMultiline("import", {
+body: "ok" });
```

Comment:
Could we get a test case with a proper import statement that fails without this PR?
IIUC, it should be something like:
```
import { something } from
'package'
```
Or something based on the issue this is fixing.